### DISCUSSION TITLE
box styles without top/bottom rules still pad with a line space

### DISCUSSION THREAD

**Discussion by mcpherson** [Textualize/rich#1430] [Ideas] [answered]
I'm playing around trying to minimize my printed information (in lines). I find that if I use (say, for a `Table`) `box.SIMPLE`, there are still two blank lines where the top and bottom rules would be on more complex `box`es (e.g. `box.SQUARE`). See the attached images showing the difference.

It seems to me that, for maximum control, there should at least be an option to remove those extra blank lines (to tighten up vertical spacing if desired). Just throwing it out there for discussion.

In any case, I really dig the package--thanks for putting it out!

![Screen Shot 2021-08-24 at 5 01 30 PM](https://user-images.githubusercontent.com/583056/130701738-a92041ea-e217-4aac-8c4b-0614c64d0952.png)

![Screen Shot 2021-08-24 at 5 02 13 PM](https://user-images.githubusercontent.com/583056/130701747-46e6daf1-7491-48d3-bbf5-f7364d3d5877.png)

**Comment by willmcgugan** [comment, marked as answer]
Add `show_edge=False` to your Table.

**Reply by mcpherson** [reply]
Thanks! Saw that option but thought `box` would override those values. My bad.
